package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ml-testing-toolkit-ui",
-  "version": "10.3.0",
+  "version": "10.3.3",
```

Comment:
why 10.3.3 and not 10.3.1?

Reply:
The backend version is at 10.3.3, so just wanted to sync with it.